src/wp-includes/meta.php

```diff
@@ -1451,6 +1454,9 @@ function register_meta( $object_type, $meta_key, $args, $deprecated = null ) {
 	 * @param string $meta_key    Meta key.
 	 */
 	$args = apply_filters( 'register_meta_args', $args, $defaults, $object_type, $meta_key );
+	if ( isset( $args['label'] ) && ! isset( $args['show_in_rest'] ) ) {
```

Comment:
This should change before the filter so 3rd party code can still change it.

Reply:
What happens when the filter adds a label without setting `show_in_rest`? Shouldn't that follow the same behavior?

Reply:
Maybe it is safer to move it before the filter as you say to ensure people can change this behavior.